### ISSUE TITLE
UnnecessaryMantissa should not check URLs

### ISSUE THREAD

**Issue by laurent-le-graverend** [brigade/scss-lint#772]
Getting this error:
```
[W] UnnecessaryMantissa: `.0` should be written without the mantissa as ``
```

For the following class:
```
.something {
  background: url(https://www.example.com/v1.0/image.jpg);
}
```

`v1.0` should not raise an error in this case.
